stat/distuv/chi.go

```diff
@@ -0,0 +1,111 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
```

Comment:
Maybe also add Entropy https://en.wikipedia.org/wiki/Chi_distribution#Entropy and Median ?

Reply:
No good deed goes unpunished.